### ISSUE TITLE
performance on UCF and HMDB (split1) 

### ISSUE THREAD

**Issue by lovelyczli** [facebookresearch/VMZ#125]
Thanks for sharing exciting work! But I have some questions as follow:

UCF and HMDB are popular datasets for action recognition, but your work doesn't report these results. (except R2+1D report pre-training on Kinetics)
I3D has greatly improved on HMDB and UCF since pre-training on Kinetics. So how about R2+1D and CSN after pre-training on big datasets, especially in IG65M.

I am attempting to finetune on UCF and HMDB by your Initialization weights by Kinetics and IG65M. So can you provide me some experimental results?
Thanks

**Comment by salahsh** [comment]
I have trained r2plus1d_18 on HMDB51, but the accuracy is less than 40%, do not know why
